lib/main.dart

```diff
@@ -1,23 +1,26 @@
 import 'package:flutter/material.dart';
```

Comment:
Remove main.dart from models.it should be outside of all layers in the lib package